wiki/Glossary/en.md

```diff
@@ -547,16 +542,16 @@ M4M, or _Mod for Mod_, is a term describing when a modder agrees to mod another
 
 ### Marathon
 
-A marathon is a type of beatmap that can only be approved.
+A marathon is a type of [beatmap](/wiki/beatmap) that can only be approved.
 
 Marathon beatmaps are considered to be, at least, 6 minutes long and can only have one mapped difficulty.
```

Comment:
Remove commas:
`Marathon beatmaps are considered to be at least 6 minutes long and can only have one mapped difficulty.